src/form-data-wrapper.js

```diff
@@ -127,3 +127,13 @@ export class FormDataWrapper {
     return this.formData_;
   }
 }
+
+/**
+ * Check if the given object is a FormDataWrapper instance
+ * @param {*} o
```

Comment:
Should this be `{!Object}`?

Reply:
Probably, but Closure Compiler complains that `init.body` is `(Object|string|undefined)`, but I don't see why we'd make that the `@param` type. What do you think?

Reply:
This is fine with the truthy check you added. Still think that adding `typeof` is a bit more precise (and has precedence).

Reply:
Coo, I just saw that. I like that better too